pontoon/base/views.py

```diff
@@ -1075,9 +1074,8 @@ def transvision(request, repo, title):
         log.error(str(e))
         return HttpResponse("error")
 
-    src = "en-US"
-    url = "https://transvision.mozfr.org/api/v1/tm/%s/%s/" \
-          "%s/%s/?max_results=%s&min_quality=70" % (repo, src, locale, text, 5)
+    url = "https://transvision.mozfr.org/api/v1/tm/global/en-US/" \
```

Comment:
`str.format` is preferred over `%` formatting. Now's a good time to update this.

Also, wrapping strings in parenthesis and relying on adjacent strings being concatenated is preffered over using `\` to escape the line break, as it's less fragile to changes.